### PR TITLE
Do not run the Gradle tests when using -DquicklyDocs and add some useful workflow tips

### PR DESCRIPTION
**Pull request by gsmet**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -409,6 +409,52 @@ Thus, it is recommended to use the following approach:
...
+- If using plain Maven
+
+```sh
+alias build-fast="mvn -T0.8C -e -DskipDocs -DskipTests -DskipITs -Dinvoker.skip -DskipExtensionValidation -Dskip.gradle.tests -Dtruststore.skip clean install"
+alias build-docs="mvnd -T0.8C -e -DskipTests -DskipITs -Dinvoker.skip -DskipExtensionValidation -Dskip.gradle.tests -Dskip.gradle.tests -Dtruststore.skip -Dno-test-modules -Dasciidoctor.fail-if=DEBUG clean install"
+alias format="mvnd -T0.8C process-sources -Denforcer.skip -Dprotoc.skip"
+alias qss="java -jar ${HOME}/git/quarkus/devtools/cli/target/quarkus-cli-999-SNAPSHOT-runner.jar"
+```
+
+Using `./mvnw` is often not practical in this case as you might want to call these aliases from a nested directory.
```
Comment (maxandersen) [inline CONTRIBUTING.md:456]: I tend to use gum for solving that issue.

https://andresalmiray.com/gum-the-gradle-maven-wrapper/

then have alias mvn=gm which then locates ./mvnw if in root.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -409,6 +409,52 @@ Thus, it is recommended to use the following approach:
...
+- `build-fast`: build the Quarkus artifacts and install them
+- `build-docs`: run from the root of the project, build the documentation
+- `format`: format the source code following our coding conventions
+- `qss`: run the Quarkus CLI from a snapshot (make sure you build the artifacts first)
+
+- If using mvnd
+
+```sh
+alias build-fast="mvnd -e -DskipDocs -DskipTests -DskipITs -Dinvoker.skip -DskipExtensionValidation -Dskip.gradle.tests -Dtruststore.skip clean install"
+alias build-docs="mvnd -e -DskipTests -DskipITs -Dinvoker.skip -DskipExtensionValidation -Dskip.gradle.tests -Dskip.gradle.tests -Dtruststore.skip -Dno-test-modules -Dasciidoctor.fail-if=DEBUG clean install"
```
Comment (brunobat) [inline CONTRIBUTING.md:442]: -Dskip.gradle.tests is duplicated

Comment (gsmet) [reply CONTRIBUTING.md:442]: Yeah I really want to skip them :]

I'll fix it.

**Comment by brunobat** [comment]
I tried the `mvnd` quick-build and seems ok, It really cuts the build time if you run it multiple times. 
It requires mvnd 1.0.2+, with the previous version it will fail.

**Review by FroMage — APPROVED** [review]
Nice aliases, thanks!

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -409,6 +409,52 @@ Thus, it is recommended to use the following approach:
...
+alias format="mvnd process-sources -Denforcer.skip -Dprotoc.skip"
+alias qss="java -jar ${HOME}/git/quarkus/devtools/cli/target/quarkus-cli-999-SNAPSHOT-runner.jar"
+```
+
+- If using plain Maven
+
+```sh
+alias build-fast="mvn -T0.8C -e -DskipDocs -DskipTests -DskipITs -Dinvoker.skip -DskipExtensionValidation -Dskip.gradle.tests -Dtruststore.skip clean install"
```
[the remark this answers is not in the document]
Comment (gsmet) [reply CONTRIBUTING.md:450]: Conventions are not ours. They are coming from the Maven plugins we are using (except `skip.gradle.tests` which is indeed not ideal).

[the remark this answers is not in the document]
Comment (FroMage) [reply CONTRIBUTING.md:450]: I figured :(

**Comment by gsmet** [comment]
I applied all your suggestions.

**Pull request merged**